### ISSUE TITLE
Error with tap

### ISSUE THREAD

**Issue by jedahan** [hmarr/homebrew-tap#1]
```shell
 ▲ brew tap hmarr/tap                                               ⇡ macos 39d
==> Tapping hmarr/tap
Cloning into '/opt/homebrew/Library/Taps/hmarr/homebrew-tap'...
remote: Enumerating objects: 30, done.
remote: Counting objects: 100% (30/30), done.
remote: Compressing objects: 100% (19/19), done.
remote: Total 30 (delta 7), reused 17 (delta 5), pack-reused 0
Unpacking objects: 100% (30/30), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/hmarr/homebrew-tap/Formula/codeowners.rb
formulae require at least a URL
Error: Cannot tap hmarr/tap: invalid syntax in tap!
```

**Comment by hmarr** [comment]
Thanks for reporting this. It looks like none of the builds are compatible with your computer. Are you using one of the new ARM-based M1 Macs by any chance? If so, I think this can be fixed by compiling for ARM too.

**Comment by hmarr** [comment]
I just released v0.3.2, which [includes](https://github.com/hmarr/homebrew-tap/blob/ac13422cb52477d5e6ed1c8dba4679c540325706/Formula/codeowners.rb#L15-L18) a build for Apple silicon. 

Can you try it again and let me know if it succeeds?

**Comment by jedahan** [comment]
`brew tap hmarr/tap` and `brew install vitals` both work, thanks!

p.s. yes I am on an m1
